lib/factory_bot_rails/railtie.rb

```diff
@@ -31,6 +32,22 @@ class Railtie < Rails::Railtie
       end
     end
 
+    config.after_initialize do
+      FactoryBot::SyntaxRunner.include ActiveSupport::Testing::FileFixtures
+
+      ActiveSupport.on_load :active_support_test_case do
+        FactoryBot::SyntaxRunner.file_fixture_path = file_fixture_path
+      end
+
+      if defined?(RSpec)
+        RSpec.configure do |config|
+          if config.respond_to?(:file_fixture_path)
```

Comment:
Unfortunately, this conditional is always resolving to `false`, and so the `file_fixture_path` is never set.

As of [rspec-rails@5.0.0](https://github.com/rspec/rspec-rails/blob/main/Changelog.md#500--2021-03-09), that setting is available. The current `Gemfile.lock` that are generated for me locally depends on `rspec-rails@6.0.0`, so a hunch tells me that a loading order issue is at the root of the failure.